app/models/fuel/post.rb

```diff
@@ -47,5 +48,11 @@ def teaser_content
       teaser.present? ? teaser : content
     end
 
+    def to_html
+      markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML,
+          :autolink => true, :space_after_headers => true)
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.